src/aap_eda/settings/default.py

```diff
@@ -133,14 +133,16 @@ def _get_secret_key() -> str:
 
 # JWT token lifetime
 JWT_ACCESS_TOKEN_LIFETIME_SECONDS = settings.get(
-    "JWT_ACCESS_TOKEN_LIFETIME_SECONDS", 5
```

Comment:
I think the task:format will do this one line format instead of 2 lines.

Reply:
it depends on the final comma. My env with rust -all, always suggests the trailing comma but we are not enforcing it in CI.